toolkit/imageconfigs/packagelists/isoinstaller-64k-packages.yaml

```diff
@@ -0,0 +1,71 @@
+packages:
+- pam
```

Comment:
Are these packages in a particular order? if not can they be listed alphabetically?

Reply:
Updated